integration/testrunner/grpcrunner/grpc.go

```diff
@@ -252,13 +292,21 @@ func createConnection(tb testing.TB, addr string, idToken string) *grpc.ClientCo
 // This query is used to find the relevant audit log in BigQuery, which we assume will be added by the server.
 // We specifically look up the log using the UUID specified in the request as we know the server will add that
 // as the resource name, and provides us a unique key to find logs with.
-func (g *GRPC) makeQueryForGRPCUnary(u uuid.UUID) *bigquery.Query {
-	queryString := fmt.Sprintf("SELECT count(*) FROM %s.%s WHERE jsonPayload.resource_name=? LIMIT 1", g.ProjectID, g.DatasetQuery)
```

Comment:
I know this is already in the codebase, but what is the goal with `SELECT COUNT LIMIT 1`? An `EXISTS` query would be much faster.

Reply:
Yeah, so these bubble up to the same method for checking, which just checks the value of count vs expected. That way, we can use the same method for the streaming api and unary. However, you're right, the unary version could be sped up by using exists. I'll make a backlog item about it

Reply:
https://github.com/abcxyz/lumberjack/issues/263